src/model/Me.ts

```diff
@@ -79,6 +84,13 @@ export default class Me extends User {
     return new Result(new Me(result.data)); // Account API does not return a Result
   }
 
+  async phone(): Promise<PhoneVerificationResponse> {
+    const { api_url } = getConfig();
+
+    const result = await request(`${api_url}${url}/phone`, "POST");
```

Comment:
Here you can just `return request(...)` and remove the async from the signature.